### DISCUSSION TITLE
Chocolatey package has issues

### DISCUSSION THREAD

**Discussion by BradKML** [strawberrymusicplayer/strawberry#683] [Q&A] [answered]
Chocolate-coated strawberry is currently spoiled. Please replace it with a new pack.
https://community.chocolatey.org/packages/strawberrymusicplayer

**Comment by jonaski** [comment, marked as answer]
I think you need to report this to Chocolatey as we do not maintain the package.

**Reply by BradKML** [reply]
https://github.com/chocolatey-community/chocolatey-package-requests/issues/739
Something related to this?
